### PR TITLE
Add mobile ads cookbook

### LINKED ISSUE #9671: Add the Ads Recipe
Transfer the cookbook recipe from the [Google doc](https://docs.google.com/document/d/1lN2SdH6Shk5cK1PM5QWZD02wtdka6G30yJyyVTb5pls/edit#heading=h.wogef62fnza5) to the repo.

### PR DESCRIPTION
**Pull request by filiph**
Ads a cookbook recipe about `google_mobile_ads`. Applicable for both apps and games.

Fixes #9671.

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
PTAL @atsansone

**Comment by filiph** [comment]
Update: I learned about the predefined URL for this cookbook: `/cookbook/games/google-mobile-ads`. I still think this recipe's canonical place is in plugins. In contrast to other recipes that are in flight, this one isn't tied to games in any way. 

BUT: It _also_ makes sense in games, so I added a 301 redirect from `/cookbook/games/google-mobile-ads` to `/cookbook/plugins/google-mobile-ads`. 

Hope that's okay.

**Comment by parlough** [comment]
That makes sense to me, thanks Filip!

File: src/cookbook/plugins/google-mobile-ads.md [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+     ),
+   );
+
+   // Start loading.
+   bannerAd.load();
+ }
+}
+```
+
+{{site.alert.note}}
```
Comment (parlough) [inline src/cookbook/plugins/google-mobile-ads.md:397]:
```suggestion
{{site.alert.tip}}
```

File: src/cookbook/plugins/google-mobile-ads.md [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+  In many cases, you   will want to load the ad _outside_ a widget.
```
Comment (parlough) [inline src/cookbook/plugins/google-mobile-ads.md:398]:
```suggestion
  In many cases, you will want to load the ad _outside_ a widget.
```

**Review by parlough — APPROVED** [review]
Thanks for this @filiph! This is exactly what I imagine a cookbook recipe should be.

A few minor fixes and suggestions :)

**Comment by zoeyfan** [comment]
FYI. The URL will be: 
https://docs.flutter.dev/cookbook/games/google-mobile-ads

**Pull request merged**